### PR TITLE
docs(formatter): update with new options

### PR DESCRIPTION
**Pull request by sambacha**
add new options  
amend with additional examples

### CODE REVIEW

File: src/reference/config/formatter.md [hunk 1]
```diff
@@ -2,6 +2,161 @@
...
+
+The `single_line_statement_blocks` configuration option in Foundry controls whether statement blocks (code blocks enclosed in `{ }`) will be formatted to a single line or multiple lines.
+
+It has three possible values:
+
+-   `"single"` - Statement blocks will be formatted to a single line if possible.
+    
+-   `"multi"` - Statement blocks will always be formatted to multiple lines.
+    
+-   `"preserve"` - Keep the existing single/multi line formatting of statement blocks.
```
Comment (onbjerg) [inline src/reference/config/formatter.md:17]: nit: remove the additional newline between these

File: src/reference/config/formatter.md [hunk 2]
```diff
@@ -2,6 +2,161 @@
...
+
+The `multiline_func_header` configuration option in Foundry controls how function headers (the line containing the function name, parameters, return values etc.) are formatted when they exceed the max line length.
+
+Style of multiline function header in case it doesn't fit in one line. Valid possible values:
+
+-   `"attributes_first"` - (default)  Write function attributes multiline first
+
+-   `"params_first"` - Break the function header into multiple lines, with each parameter on its own line. The function name stays on the first line.
+    
+-   `"all"` - If function parameters or attributes are multiline, multiline everything
```
Comment (onbjerg) [inline src/reference/config/formatter.md:53]: nit: remove the additional newline between these

File: src/reference/config/formatter.md [hunk 3]
```diff
@@ -2,6 +2,161 @@
...
+
+From the code snippets provided:
+
+-   In `fmt.rs`, `sort_imports` is a configuration option that can be set to `true` or `false`.
+    
+-   In `formatter.rs`, the `sort_imports()` method is called to sort import statements alphabetically if `sort_imports` is enabled.
+    
+-   It finds import groups separated by blank lines in the source code.
+    
+-   Then sorts the import statements within each group alphabetically.
```
Comment (onbjerg) [inline src/reference/config/formatter.md:100]: nit: remove the additional newline between these

**Review by onbjerg — CHANGES_REQUESTED** [review]
lgtm otherwise

**Review by mattsse — APPROVED** [review]

**Comment by blowme8888** [comment]
I

**Pull request merged**